modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc

```diff
@@ -56,8 +57,8 @@ You can run the latency tests using a custom test image and image registry using
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
 -e IMAGE_REGISTRY="<custom_image_registry>" \
 -e CNF_TESTS_IMAGE="<custom_cnf-tests_image>" \
--e FEATURES=performance \
-registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} /usr/bin/test-run.sh
+-e LATENCY_TEST_RUNTIME=<time in seconds> \
+registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} /usr/bin/test-run.sh  --ginkgo.v
```

Comment:
Is the double space intentional before --ginkgo.v intentional?

Reply:
not intentional. Thanks.